tests/integration/sdb/test_vault.py

```diff
@@ -241,6 +252,24 @@ def test_sdb_kv2(self):
         )
         self.assertEqual(get_output, "bar")
 
+    @flaky
```

Comment:
You're marked this test as `flaky`. Is the test initially designed to be unstable?

Reply:
It's a copy of the surrounding functions. The tests launch a vault docker image. It generally works for me, but the whole class is marked slow and flaky, so I'm guessing there were issues with it in the past.